Shared/Localizations/ru.lproj/Localizable.strings

```diff
@@ -113,13 +114,13 @@
 // MARK: - SourcesViewController - Add Sources
 "SOURCES_VIEW_ADD_SOURCES_ALERT_TITLE" = "Добавить источник";
 "SOURCES_VIEW_ADD_SOURCES_ALERT_DESCRIPTION" = "Добавить URL репозитория AltStore";
-"SOURCES_VIEW_ADD_SOURCES_ALERT_BUTTON_IMPORT_REPO" = "Import Repositories";
-"SOURCES_VIEW_ADD_SOURCES_ALERT_BUTTON_EXPORT_REPO" = "Export Repositories";
-"SOURCES_VIEW_ADD_SOURCES_ALERT_BUTTON_EXPORT_REPO_ACTION_SUCCESS" = "Copied Repositories to Clipboard";
+"SOURCES_VIEW_ADD_SOURCES_ALERT_BUTTON_IMPORT_REPO" = "Импорт репозиториев";
+"SOURCES_VIEW_ADD_SOURCES_ALERT_BUTTON_EXPORT_REPO" = "Экспорт репозиториев";
+"SOURCES_VIEW_ADD_SOURCES_ALERT_BUTTON_EXPORT_REPO_ACTION_SUCCESS" = "Репозитории скопированы в буфер обмена";
 //Footer Validation
-"SOURCES_VIEW_ADD_SOURCES_FOOTER_NOTSTARTED" = "Enter a URL to start validation.";
-"SOURCES_VIEW_ADD_SOURCES_FOOTER_NOTVALIDJSON" = "Invalid JSON, please check your input.";
-"SOURCES_VIEW_ADD_SOURCES_FOOTER_VALID" = "Valid JSON entered.";
+"SOURCES_VIEW_ADD_SOURCES_FOOTER_NOTSTARTED" = "Введите URL чтобы начать проверку.";
```

Comment:
where is the comma 😭

Reply:
fym?

Reply:
in the sentence "Введите URL чтобы начать проверку",, a comma is needed before "чтобы" because it introduces a subordinate clause that explains the purpose or reason for the action in the main clause

"чтобы" is a conjunction used to express the goal or purpose of an action, and subordinate clauses like this are typically separated by a comma from the main clause. so the correct sentence should be "Введите URL, чтобы начать проверку."

It's kinda small detail, but still affects the overall localization

Reply:
oh, alright, thanks for clarifying